specs/latest/2.0/index.html

```diff
@@ -699,12 +700,18 @@ <h2 class="no-toc">Table of contents</h2>
   const GLenum MAX_CLIENT_WAIT_TIMEOUT_WEBGL                 = 0x9247;
 
   /* Buffer objects */
+  void bufferData(GLenum target, ByteBuffer? data, GLenum usage,
+                  optional unsigned long byteOffset = 0);
+  void bufferSubData(GLenum target, GLintptr offset, ByteBuffer data,
+                     optional unsigned long byteOffset = 0);
```

Comment:
Note: the "size" argument's semantics can be something like: if 0, then the input ArrayBuffer's or ArrayBufferView's size is used.